packages/cconv/cconv.0.4/opam

```diff
@@ -0,0 +1,33 @@
+opam-version: "1.2"
+maintainer: "simon.cruanes.2007@m4x.org"
+authors: "Simon Cruanes"
+homepage: "https://github.com/c-cube/cconv/"
+bug-reports: "https://github.com/c-cube/cconv/issues/"
+doc: "http://cedeela.fr/~simon/software/cconv/"
+tags: ["conversion" "gadt" "serialization"]
+dev-repo: "https://github.com/c-cube/cconv.git"
+build: [
+  [
+    "./configure"
+    "--%{yojson:enable}%-yojson"
+    "--%{bencode:enable}%-bencode"
+    "--%{sexplib:enable}%-sexp"
+    "--%{ppx_deriving:enable}%-ppx"
+    "--disable-tests"
+    "--%{doc:enable}%-docs"
+  ]
+  [make "build"]
+]
+install: [make "install"]
+build-test: [make "test"]
+build-doc: [make "doc"]
+remove: ["ocamlfind" "remove" "cconv"]
+depends: [
+  "cppo" {build}
+  "ocamlfind" {build}
+]
+depopts: ["bencode" "sexplib" "yojson" "ppx_deriving"]
+conflicts: [
+  "ppx_deriving" {< "2.0"}
+]
+available: [ocaml-version >= "3.12"]
```

Comment:
Should this be `>= "4.01.0"` (to match the [`_oasis` file](https://github.com/c-cube/cconv/blob/6ea622a36aaa31c20e74d4ddecdd52009e4280ae/_oasis#L11))?